app/src/main/java/com/example/golocal/fragments/ProfileFragment.java

```diff
@@ -12,33 +19,43 @@
 import android.view.ViewGroup;
 import android.widget.Button;
 import android.widget.EditText;
+import android.widget.ImageButton;
 import android.widget.ImageView;
 import android.widget.TextView;
+import android.widget.Toast;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+import androidx.core.content.FileProvider;
 import androidx.fragment.app.Fragment;
 
 import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.target.Target;
 import com.example.golocal.R;
 import com.example.golocal.activities.LoginActivity;
 import com.parse.ParseException;
 import com.parse.ParseFile;
 import com.parse.ParseUser;
 import com.parse.SaveCallback;
 
+import java.io.File;
+
 public class ProfileFragment extends Fragment {
     private TextView tvUsername;
     private TextView tvBio;
     private ImageView ivProfileImage;
     private TextView tvEditBio;
     private EditText etBio;
     private Button btFinishEditBio;
+    private ImageButton ibEditProfileImage;
     private ParseUser user;
     private final String KEY_BIO = "bio";
     private final String KEY_IMAGE = "profileImage";
     private Context context;
     private final String TAG = "ProfileFragment";
+    public static final int CAPTURE_IMAGE_ACTIVITY_REQUEST_CODE = 42;
+    private File photoFile;
+    public String photoFileName = "photo.jpg";
```

Comment:
Can we reorganize these? Something like

```suggestion
public class ProfileFragment extends Fragment {
    public static final int CAPTURE_IMAGE_ACTIVITY_REQUEST_CODE = 42;
    public String photoFileName = "photo.jpg";
    
    private final String TAG = "ProfileFragment";
    private final String KEY_BIO = "bio";
    private final String KEY_IMAGE = "profileImage";
    
    private TextView tvUsername;
    private TextView tvBio;
    private ImageView ivProfileImage;
    private TextView tvEditBio;
    private EditText etBio;
    private Button btFinishEditBio;
    private ImageButton ibEditProfileImage;
    private ParseUser user;
    private Context context;
    private File photoFile;
```